manifests/php.yml

```diff
@@ -420,6 +420,7 @@ tests/:
     test_context_propagation.py:
       Test_Otel_Context_Propagation_Default_Propagator_Api: incomplete_test_app (endpoint not implemented)
   parametric/:
+    test_process_discovery.py: missing_feature
```

Comment:
Keys must be ordered, this line should go few lines below

Reply:
is there a tool to automate that?

Reply:
Unfortunately not 😞 

It in our backlog, but lack of ressource ... #1722

Reply:
🫠 I'll write the tool tomorrow then.

Reply:
There are a ton of off the shelf tools for this, eg https://github.com/marketplace/actions/yaml-lint

For the above, we could use mostly default settings and enable the key-ordering rule
https://yamllint.readthedocs.io/en/stable/configuration.html#default-configuration

Reply:
That would be awesome ❤️ . I've precised in the issue what to do